apps/hexagon_dma/process_nv12.cpp

```diff
@@ -47,11 +47,19 @@ int main(int argc, char **argv) {
     // The Last parameter 0 indicate DMA Read
     halide_hexagon_dma_prepare_for_copy_to_host(nullptr, input, dma_engine, false, eDmaFmt_NV12);
 
-    Halide::Runtime::Buffer<uint8_t> output(width, height, 2);
+    //Halide::Runtime::Buffer<uint8_t> output(width, height, 2);
+    Halide::Runtime::Buffer<uint8_t> output(width, (3 * height) / 2);
+    output.embed(2, 0);
```

Comment:
This (and the following 3 lines) should be done to the chroma buffer only, after cropping it out of the outer buffer)